### ISSUE TITLE
Подскажите пож. с какой минимальной версии Wordpress можно использовать данный плагин и какая версия наилучшим образом подходит.

### ISSUE THREAD

**Issue by anpilov2007** [GdeSlon/wp-affiliate-shop#31]
Подскажите пож. с какой минимальной версии Wordpress можно использовать данный плагин и какая версия наилучшим образом подходит.

**Comment by letions** [comment]
Здравствуйте! Помогу загрузить товары с прайс-листа на OpenCart или Ostore — смотрятся сайты намного солиднее на данных CMS. Вот пример — http://myopencart.com/ .
В карточке товара будут помимо основного фото — дополнительные фото, актуальная цена, заголовок, наличие товара, описание, характеристики с атрибутами — все как на сайте рекламодателя!
Создам интернет-магазин с Вашим прайсом.
Пишите на letions@mail.ru

**Comment by fishkin40** [comment]
По всей видимости лучше не использовать совсем.

**Comment by fishkin40** [comment]
Да, проверено! Ребята забейте на слона! Прибыли нет!!!!!!!!!!!!!!!
